src/Viewer.tsx

```diff
@@ -325,32 +377,6 @@ function Viewer(): ReactElement {
     }
   }, []);
 
-  /**
-   * Resets the color ramp to a default min and max value based on the feature and dataset.
-   *
-   * If the feature is thresholded, the color ramp will be set to the threshold's min and max.
-   * Otherwise, the color ramp will be set to the feature's min and max.
-   *
-   * (Does nothing if the viewer is configured to keep the range between datasets.)
-   */
-  const resetColorRampRangeToDefaults = useCallback(
-    (featureDataset: Dataset, featureKey: string): void => {
-      const featureData = featureDataset.getFeatureData(featureKey);
```

Comment:
Only change was the removal of this line, since `featureData` is already stored as state.